src/com/raghav/javabasics/day4/FindHighestOccurredCharInString.java

```diff
@@ -0,0 +1,39 @@
+package com.raghav.javabasics.day4;
+// Given a string, write a program to return the highest occurred character in a String.
+public class FindHighestOccurredCharInString {
+    public static void main(String[] args) {
+        FindHighestOccurredCharInString result = new FindHighestOccurredCharInString();
+        result.charCount("abc");
```

Comment:
Output:
MAX occurred char is: a - 1, MAX occurred char is: b - 1, MAX occurred char is: c - 1, 
Array is Empty/Null
MAX occurred char is: c - 5, 
MAX occurred char is: $ - 3, MAX occurred char is: 1 - 3, MAX occurred char is: 2 - 3, MAX occurred char is: 3 - 3, 
MAX occurred char is: a - 2, MAX occurred char is: b - 2, MAX occurred char is: d - 2, 
Array is Empty/Null